### PR TITLE
Rename README and add titles

### LINKED ISSUE #1: Use markdown in README
https://guides.github.com/features/mastering-markdown/

### PR DESCRIPTION
**Pull request by nahueespinosa**
Closes #1

### CODE REVIEW

**Comment by marcosmarp** [comment]
everything ok

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# SignIn-SignUp project
+
+## Description
 Simple data base program with registration and log in system. 
 
-For more info feel free to contact me on github: https://github.com/marcosmarp
\ No newline at end of file
+## More info
```
Comment (nahueespinosa) [inline README.md:6]: Nice title!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# SignIn-SignUp project
```
Comment (nahueespinosa) [inline README.md:1]: Ok

**Review by nahueespinosa — COMMENTED** [review]
Good job!

**Review by marcosmarp — APPROVED** [review]
Done

**Pull request merged**